portfolio/src/main/webapp/script.js

```diff
@@ -11,19 +11,81 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+const imgFilePath = '/images/album/';
+const images = [ 
+  {
+    'filePath': `${imgFilePath}LoveIsABeautifulThing_Vulfpeck.jpg`,
```

Comment:
instead of treating this as a dictionary (string keys) let's treat it as a real object! leave out the quotes here. In the accessors below use the dot notation; so something like:

images[i].filePath